tests/forms_tests/tests/test_forms.py

```diff
@@ -5329,3 +5386,31 @@ class CustomFrameworkForm(FrameworkForm):
             '<label for="id_name" class="required">Name:</label>'
             '<legend class="required">Language:</legend>',
         )
+
+    def test_custom_boundfield(self):
```

Comment:
I like that this test checks that the field option is given priority over the form one. 

Please can the test be updated to show that the form option is given priority over the project-wide option from a custom `FORM_RENDERER`.

Reply:
@christophehenry Was this added to the PR? I'm not sure I'm finding it.

Reply:
Yep, [it's here](https://github.com/django/django/pull/18266/files#diff-c0735e6cb7a54da7042230dc30a971119e5ad40cdb47cdeaa1574c7022f4c0feR5382). I reshaped thoses tests a bit to avoid duplication.